### ISSUE TITLE
Performance on scannet

### ISSUE THREAD

**Issue by smileyenot983** [Pointcept/PointTransformerV3#138]
Hi everyone! Did anyone have any issues with the metrics on scannet validation dataset? I am using `semseg-pt-v3m1-0-base.py` config with `scannet-semseg-pt-v3m1-0-base` weights, scannet was taken from scannet-preprocessed and am having issues with the quality. I did not yet finish running it on the whole scannet dataset, but saw that quality(mIoU) is much lower than it was reported in the paper.

Here are my logs:

[2025-03-02 19:03:33,107 INFO test.py line 207 3475] Test: 66/312-scene0591_02, Batch: 150/151
[2025-03-02 19:03:33,253 INFO test.py line 295 3475] Test: scene0591_02 [66/312]-187656 Batch 18.934 (16.378) Accuracy 0.4881 (0.3356) mIoU 0.1384 (0.2026)

Inside the parenthesis there is a mIou if i am not mistaken and it is equal to 0.2026, but in the paper it reaches 77.5(or 78.6). 

Did someone has similar issues? Or did I run it the wrong way?

**Comment by smileyenot983** [comment]
I found out following statement:
*Released model weights are temporarily invalid as the model structure of PTv3 is adjusted.

Is this still true? If so, when should we expect the release?

**Comment by Gofinge** [comment]
Hey, as an individual contributor, I can't maintain the model weight for each version of the codebase. Maybe you can train the model by yourself, or try our latest self-supervised pre-trained PTv3: https://github.com/facebookresearch/sonata
